tools/topology/sof/pipe-processing-playback.m4

```diff
@@ -48,15 +48,6 @@ C_CONTROLBYTES(Post Process Setup Config, PIPELINE_ID,
 	,
 	PP_SETUP_CONFIG)
 
-
-# Post process runtime params
```

Comment:
Why do you want to remove this placeholder for runtime params?

Reply:
Aren't the params in the blob above? These aren't linked in anywhere

Reply:
If these are used then we should add a comment that they are implicitly linked in because right now grepping for `PP_RUNTIME_PARAMS` yields only the result below

Reply:
I also feel the removal is unwarranted, how do you communicate info that isn't part of the regular parameters (sample rate, bit width etc)? In case one of the codecs will want this.

Reply:
There is a blob above I assumed was communicating that since this one appears to unconnected to the actual component.

Reply:
I see this as a unused variable without an explanation, please explain why you think it should be kept. It either needs an explanation or a usage.

Reply:
@cujomalainey, this processing topology is missing associated `C_CONTROLBYTES` which I am going to add in the next PR. But in principle this is a placeholder for runtime params so one can use it to pass bespoke runtime parameters using sof-ctl. This can also be used to provide default runtime params, so I this shouldn't be removed.

Reply:
But the params are not passed without the `C_CONTROLBYTES` no? If that is all that is missing then that satisfies usage imo.

Reply:
Will drop this commit, @mrajwa can you link the PR you will fix this in?

Reply:
yes, I will once pushed.